Library/Formula/symlinks.rb

```diff
@@ -0,0 +1,23 @@
+class Symlinks < Formula
+  desc "scan/change symbolic links"
+  homepage "http://www.ibiblio.org/pub/Linux/utils/file/symlinks.lsm"
```

Comment:
This is unlikely to happen, but even if it did, `--env=std` also makes sure to pass `-isysroot` and the build system (`make` with its implicit rules) shouldn't be hostile enough to ignore that.